src/wp-includes/link-template.php

```diff
@@ -4260,24 +4261,21 @@ function the_shortlink( $text = '', $title = '', $before = '', $after = '' ) {
 		$text = __( 'This is the short link.' );
 	}
 
-	if ( empty( $title ) ) {
-		$title = the_title_attribute( array( 'echo' => false ) );
-	}
-
 	$shortlink = wp_get_shortlink( $post->ID );
 
 	if ( ! empty( $shortlink ) ) {
-		$link = '<a rel="shortlink" href="' . esc_url( $shortlink ) . '" title="' . $title . '">' . $text . '</a>';
+		$link = '<a rel="shortlink" href="' . esc_url( $shortlink ) . '">' . $text . '</a>';
 
 		/**
 		 * Filters the short link anchor tag for a post.
 		 *
 		 * @since 3.0.0
+		 * @since 6.8.0 Removed title attribute.
 		 *
 		 * @param string $link      Shortlink anchor tag.
 		 * @param string $shortlink Shortlink URL.
 		 * @param string $text      Shortlink's text.
-		 * @param string $title     Shortlink's title attribute.
+		 * @param string $title     Unused.
```

Comment:
In the context of the filter, I think this parameter should say something like "Custom title attribute. Unused." For any extender wishing to retain the title attribute, this is the relevant documentation.

Reply:
"Custom" could be fine, but I just restored "Shortlink's title attribute."